Framework/Kernel/src/Json.cpp

```diff
@@ -0,0 +1,67 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+
+#include "MantidKernel/Json.h"
+#include <boost/algorithm/string/erase.hpp>
+#include <sstream>
+
+namespace Mantid {
+namespace Kernel {
+namespace JsonHelpers {
+
+/**
+ * @brief Return a string given the json value passed
+ *
+ * @param json the json to be converted to a string.
+ * @param indentation the indentation can be anything, if nothing is passed then indentations and newline charecters
+ * will be ignored.
+ * @return std::string
+ */
+std::string jsonToString(const Json::Value &json, const std::string &indentation) {
+  if (!json.isObject()) {
+    throw std::invalid_argument("Expected Json::Value of type object or string, found single non-string value type.");
+  }
+
+  Json::StreamWriterBuilder builder;
+  builder.settings_["indentation"] = indentation;
+  auto string = Json::writeString(builder, json);
+  return string;
+}
+
+/**
+ * @brief Return a Json::Value given the string passed in
```

Comment:
I like the simplicity of these wrappers :)

One quick thought. This and `parse` both say "take a string and give me back a Json::Value. How might someone determine which to use? Can we add some helpful hints to the docstrings?

Reply:
So the main difference is the interface that the developer wants to use, I am not sure what someone would use to determine it so I couldn't write anything. They largely do a similar thing but output differently and take slightly different inputs.

Reply:
Okay. That's a fair point. It might be worth commenting that to say these methods should produce the same `Json::Value` given the same input but differ on how errors are handled.

Reply:
I can do that